lib/utils/fast-boot.js

```diff
@@ -0,0 +1,22 @@
+const { getParent } = require('../utils/common');
+
+function isEnvironmentBrowser(node) {
+  let callee = node.test.callee;
+
+  if (callee) {
+    let obj = callee.object;
+    let prop = callee.property;
+
+    return obj.name === 'environment' && prop.name === 'isBrowser';
```

Comment:
I think in the long run it should be [isFastBoot](https://github.com/ember-fastboot/ember-cli-fastboot/blob/master/app/services/fastboot.js#L92). `environment.isBrowser` support will be dropped soon (TM)